components/features/housing/DutyRoster.tsx

```
@@ -139,13 +141,22 @@ export default function DutyRoster({
+            <h2 className="font-bebas text-2xl text-stone-700">
+              Master Task Roster
+            </h2>
+            <p className="text-stone-500 text-sm">
+              All active and recent tasks in the system.
+            </p>
+          </div>
+          <button
+            onClick={onRefresh}
+            className="p-2 text-stone-400 hover:text-fiji-purple transition-colors"
+            title="Refresh Roster"
+          >
```

Biome (2.4.6)

[error] Provide an explicit type prop for the button element.

(lint/a11y/useButtonType)